content/events/2024-in-person-conference/_index.md

```diff
@@ -93,6 +93,11 @@ We thank the [Software Sustainability Institute](https://www.software.ac.uk/) fo
 
 <img alt="Software Sustainability Institute logo" src="https://raw.githubusercontent.com/softwaresaved/outreach-materials/master/logos/SSI-LOGOS/RED%20LOGOS/SSI_PRIMARY-LOGO.png" style="max-width: 420px">
 
+
+We thank Aalto Scientific Computing for providing the venue and local support.
+
+<img alt="Aalto Scientific Computing logo" src="https://github.com/AaltoSciComp/scicomp-docs/blob/f0b67cb8f38a47f56dd6665b5efe0de5f33e91af/_static/asc-socialshare-02.png" style="max-width: 420px">
```

Comment:
does not seem to render, can you try the `raw.githubusercontent.com` link like before?

Reply:
It took me a while to figure out the url, but it should work now.